statsmodels/genmod/generalized_linear_model.py

```diff
@@ -1678,12 +1700,15 @@ def summary(self, yname=None, xname=None, title=None, alpha=.05):
 
         if hasattr(self, 'constraints'):
             smry.add_extra_txt(['Model has been estimated subject to linear '
-                          'equality constraints.'])
+                                'equality constraints.'])
 
-        #diagnostic table is not used yet:
-        #smry.add_table_2cols(self, gleft=diagn_left, gright=diagn_right,
-        #                  yname=yname, xname=xname,
-        #                  title="")
+        if hasattr(self, 'cov_type'):
```

Comment:
this is too late and has no effect.  (the summary instance is already filled in line 1696
it should be right after the top_right list on line 1690 before title

Reply:
That's embarrassing.